tests/pipelines/eks/small/upstream-load.yaml

```diff
@@ -56,3 +56,5 @@ spec:
       params:
       - name: cluster-name
         value: "eks-small-upstream-load-$(context.pipelineRun.uid)"
+      - name: results-bucket
+        value: "$(date +'%Y/%m/%d/%HH/%mm)/$(context.pipelineRun.uid)"
```

Comment:
Nice! The only downside of this is we now need to configure this in every one of our pipelines, which is kind of a bummer. It would be nice if the task could somehow be aware of the taskrun and use the ID there as part of the CL2 spec.